integration-tests/tests/src/utils/select.ts

```diff
@@ -24,14 +24,14 @@ type PlatformValues<T> = Partial<Record<keyof KnownEnvironment, T>> & {
  * @param platformValues containing value for each targeted platform.
  * @returns the value for the platform the test is running on.
  */
-export function select<T>(platformValues: PlatformValues<T>): T {
+export function select<T>(platformValues: PlatformValues<T>): any {
```

Comment:
Why this change? It'll be harder to determine a type mismatch in the calling code.